spring-boot-starter-camunda/src/main/java/io/camunda/zeebe/spring/client/properties/CommonClientConfigurationProperties.java

```diff
@@ -0,0 +1,146 @@
+package io.camunda.zeebe.spring.client.properties;
+
+import io.camunda.commons.auth.*;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties
```

Comment:
Could we create configuration properties as classes and not use `@Value` on each property?

This would make the code more readable and the properties classes would properly reflect the property structure of the `application.yaml`.

Reply:
Got rid of `@Value`s and refactored as its own configuration properties in https://github.com/camunda-community-hub/spring-zeebe/pull/510/commits/2a13aa5a78ea11c117eae5e356615b180113bc0a